src/associates/lib/main.css

```diff
@@ -1,14 +1,7 @@
-#su-associates button{background-color:#8c1515;color:white;height:30px;padding:0 1.5em;float:right;margin-right:50%}#su-associates .resultsContainer{margin-bottom:3em;margin-top:3em}
-
+#su-associates button{background-color:#8c1515;color:#fff;height:30px;padding:0 1.5em;float:right;margin-right:50%}#su-associates .resultsContainer{margin-bottom:3em;margin-top:3em}
 #su-associates .react-autosuggest__input{width:25%}
-
 #su-associates .newMembersToggle{margin-top:3em}
-
-#su-associates .name{margin-left:5%;padding:0.3em 0;display:inline-block;width:40%}#su-associates .degrees{margin-left:5%;display:inline}
-
-body{color:#726543;font-size:13px}.newMembersToggle{margin-top:2em;margin-left:3%}.nav-tabs .nav-item.active{background-color:#726543;color:white}.nav-tabs .nav-item{background-color:#dcdac1;color:#726543;border:1px solid #c1bda2;border-radius:0}.tab-pane{background-color:#f3f1ed;border-color:#dddbc3}#su-associates .nav{margin-top:3em}#su-associates h5{width:95%;color:#7d9a18;border-bottom:1px dashed #e0ddd6;padding:10px;margin:0 auto 20px auto}#su-associates .topLink{color:#8c1515;float:right;margin-right:2%;margin-top:-1.7em}
-
+#su-associates .name{margin-left:5%;padding:.3em 0;display:inline-block;width:40%}#su-associates .degrees{margin-left:5%;display:inline}
+body{color:#726543;font-size:13px}.newMembersToggle{margin-top:2em;margin-left:3%}.nav-tabs .nav-item.active{background-color:#726543;color:#fff}.nav-tabs .nav-item{background-color:#dcdac1;color:#726543;border:1px solid #c1bda2;border-radius:0}.tab-pane{background-color:#f3f1ed;border-color:#dddbc3}#su-associates .nav{margin-top:3em}#su-associates h5{width:95%;color:#7d9a18;border-bottom:1px dashed #e0ddd6;padding:10px;margin:0 auto 20px auto}#su-associates .topLink{color:#8c1515;float:right;margin-right:2%;margin-top:-1.7em}
```

Comment:
Looks like it just removes the empty lines here @rebeccahongsf yeah? 😄

Reply:
@yvonnetangsu oh yeah 😆